Update 1-blockchain-101.md

DavidoAprendiz opened the pull request:
1st Draft

DavidoAprendiz committed:
Update 1-blockchain-101.md

1st Draft

kii-dot committed:
added some comments and changed some stuff

DavidoAprendiz committed:
Update 1-blockchain-101.md

2nd Draft
- “What is a Block” was fixed and moved up in the document.
- Created new headers
- Added “mermaid.js style” with some explanation graphs

kii-dot commented on docs/into-the-woods/prepwork-0-what-is-blockchain/1-blockchain-101.md:
```diff
@@ -6,4 +6,122 @@ sidebar_position: 0
+
+---
+### What is a Block?
+
```
YES! Love this section

kii-dot commented on docs/into-the-woods/prepwork-0-what-is-blockchain/1-blockchain-101.md:
```diff
@@ -6,4 +6,122 @@ sidebar_position: 0
+
+A smart contract is an on-chain code that upon a successful validation, is executed on the blockchain creating a smart **transaction**.
+
+In Ergo the smart contract is called "Guard Script".
```
replace with "a smart contract is also called a Guard Script"

kii-dot commented on docs/into-the-woods/prepwork-0-what-is-blockchain/1-blockchain-101.md:
```diff
@@ -6,4 +6,122 @@ sidebar_position: 0
+### What is a Transaction(Tx)?
+A transaction(TX for short) is when an exchange of Value/Data between two or more Addresses occurs.
+
+- In the Account model, a transaction only need to have **Input**.
```
I'm not sure about this. I'll leave this out to be honest and focus entirely on eUTxO

kii-dot commented on docs/into-the-woods/prepwork-0-what-is-blockchain/1-blockchain-101.md:
```diff
@@ -6,4 +6,122 @@ sidebar_position: 0
+- In the Account model, a transaction only need to have **Input**.
+  - The Input can be the amount to send.
+- In the eUTXO model, a transaction need to have **Input** and will always have **Output**.
+  - The Output is what is considered the "change" meaning what's left in native token.
```
this statement may be misleading or wrong. 

Change sounds like what is left.

The Output are modeled boxes of the transaction.

The total value for native token and tokens in inputs has to always be equal to output. again output is not just the "change"

kii-dot commented on docs/into-the-woods/prepwork-0-what-is-blockchain/1-blockchain-101.md:
```diff
@@ -6,4 +6,122 @@ sidebar_position: 0
+
+---
+### Test your knowledge
+<details> 
```
I'd add a little more test your knowledge that really get them thinking and ideating. 

Even have some cabin session style if that helps

DavidoAprendiz committed:
3rd Draft

- Applied reviews
- Change headers to be compliance with current documentation on repo
- Changed "What is a Transaction(Tx)?"
- Added more questions in “Test your knowledge”
- Added “Conclusion” topic
- Added “Cabin Sessions”

DavidoAprendiz committed:
Update .gitignore
